README-cn.md

```diff
@@ -0,0 +1,125 @@
+# dst [![Build Status](https://travis-ci.com/jovany-wang/dst.svg?token=7wQpC1f51BkWzsCeefLE&branch=master)](https://travis-ci.com/jovany-wang/dst)
```

Comment:
Could you please request a separated PR to add a `README-cn.md`?

Reply:
okay